### PR TITLE
docs: changelog for 3.24.3

### PR DESCRIPTION
**Pull request by chenshuai2144**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

changelog for 3.24.3

## 3.24.3

`2019-10-26`

- 🐞 修复 Table `columns` 在 render 中会导致排序失效的问题。[#19404](https://github.com/ant-design/ant-design/pull/19404)
- 🐞 修复 Grid 响应式 `gutter` 失效的问题。[#19308](https://github.com/ant-design/ant-design/pull/19308)
- 🐞 修复小号 Table 边框样式问题。[#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
- 🐞 修复 Dropdown 中的 Menu 在 SubMenu 选中后父级菜单未高亮的问题。[#19313](https://github.com/ant-design/ant-design/pull/19313)
- 🐞 修复 Typography & Drawer & Statistic 中的 moment.js 无法被 tree-shaking 的问题。[#19317](https://github.com/ant-design/ant-design/pull/19317)
- 🐞 修复 Input.Search 出现 `non-boolean attribute` 和 `unique key` 警告信息。[#19357](https://github.com/ant-design/ant-design/pull/19357)
- 🐞 修复 Modal 没有标题时，图标没能对齐的问题。[#19419](https://github.com/ant-design/ant-design/pull/19419)
- 💄 修复 Checkbox 在被禁用时，光标仍为可选状态的问题。[#19403](https://github.com/ant-design/ant-design/pull/19403)
- 💄 新增 `@modal-header-color` 变量。[#19385](https://github.com/ant-design/ant-design/pull/19385)
- 🛠 多处 TypeScript 的定义更新。[#19377](https://github.com/ant-design/ant-design/pull/19377) [#19343](https://github.com/ant-design/ant-design/pull/19343) [#19333](https://github.com/ant-design/ant-design/pull/19333)

--------

- 🐞 Fix Table sort not working when `columns` inside render. [#19404](https://github.com/ant-design/ant-design/pull/19404)
- 🐞 Fix Grid responsive `gutter` not working. [#19308](https://github.com/ant-design/ant-design/pull/19308)
- 🐞 Fix small Table border style. [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
- 🐞 Fix the problem that Menu in Dropdown is not highlighted when SubMenu is selected. [#19313](https://github.com/ant-design/ant-design/pull/19313)
- 🐞 Fix the problem that moment.js in Typography & Drawer & Statistic can't be tree-shaking. [#19317](https://github.com/ant-design/ant-design/pull/19317)
- 🐞 Fix Input.Search react warnings of `non-boolean attribute` and `unique key`. [#19357](https://github.com/ant-design/ant-design/pull/19357)
- 🐞 Fix Modal icon position when not use title. [#19419](https://github.com/ant-design/ant-design/pull/19419)
- 💄 Fix a problem with the mouse pointer error when Checkbox is disabled. [#19403](https://github.com/ant-design/ant-design/pull/19403)
- 💄 Add `@modal-header-color` variable. [#19385](https://github.com/ant-design/ant-design/pull/19385)
- 🛠 Improved some TypeScript definitions. [#19377](https://github.com/ant-design/ant-design/pull/19377) [#19343](https://github.com/ant-design/ant-design/pull/19343) [#19333](https://github.com/ant-design/ant-design/pull/19333)

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/rebase_3.24.3/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/rebase_3.24.3/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+
+`2019-10-26`
+
+- 🐞 修复 Grid 响应式 `gutter` 失效的问题。 [#19308](https://github.com/ant-design/ant-design/pull/19308)
+- 🐞 修复小号 Table 边框样式问题。 [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 修复 Dropdown 父级菜单 SubMenu 的选中样式。 [#19313](https://github.com/ant-design/ant-design/pull/19313)
+- 🐞 修复 Typography & Drawer & Statistic 不能被 webpack 去除 moment.js 依赖的问题。[#19317](https://github.com/ant-design/ant-design/pull/19317)
+- 🐞 修复 Input.Search 出现 non-boolean attribute 和 unique key 警告信息。[#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 修复当没有标题时 Modal 中图标没对齐的问题。[#19419](https://github.com/ant-design/ant-design/pull/19419)
+- 💄 修复 checkbox 禁用时 cursor 错误显示错误的问题。[#19403](https://github.com/ant-design/ant-design/pull/19403)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]:
```diff
- 💄 修复 checkbox 禁用时 cursor 错误显示错误的问题。
+ 💄 修复 Checkbox 在被禁用时，光标移入后仍为可选状态的问题。
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 3.24.3
+
+`2019-10-26`
+
+- 🐞 修复 Grid 响应式 `gutter` 失效的问题。 [#19308](https://github.com/ant-design/ant-design/pull/19308)
+- 🐞 修复小号 Table 边框样式问题。 [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 修复 Dropdown 父级菜单 SubMenu 的选中样式。 [#19313](https://github.com/ant-design/ant-design/pull/19313)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]: 修复 Menu 处于 Dropdown 中、在 SubMenu 选中后父级菜单未被高亮显示的问题。

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 修复 Typography & Drawer & Statistic 不能被 webpack 去除 moment.js 依赖的问题。[#19317](https://github.com/ant-design/ant-design/pull/19317)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]: 修复 Typography & Drawer & Statistic 中的 moment.js 无法被 tree-shaking 的问题。

这个还是用术语吧，不然总觉得哪不对

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 修复 Input.Search 出现 non-boolean attribute 和 unique key 警告信息。[#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 修复当没有标题时 Modal 中图标没对齐的问题。[#19419](https://github.com/ant-design/ant-design/pull/19419)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]: 修复 Modal 没有标题时，图标没能对齐的问题。

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 💄 修复 checkbox 禁用时 cursor 错误显示错误的问题。[#19403](https://github.com/ant-design/ant-design/pull/19403)
+- 💄 新增 @modal-header-color 变量。[#19385](https://github.com/ant-design/ant-design/pull/19385)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 少 `

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 Fix small Table border style. [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 Fix Menu in Dropdown SubMenu selected style. [#19313](https://github.com/ant-design/ant-design/pull/19313)
+- 🐞 Fix Typography & Drawer & Statistic not shaking moment.js by webpack. [#19317](https://github.com/ant-design/ant-design/pull/19317)
+- 🐞 Fix Input.Search react warnings of non-boolean attribute and unique key. [#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 Fix Modal icon position when not use title. [#19419](https://github.com/ant-design/ant-design/pull/19419)
+- 💄 Fix the error cursor of the checkbox. [#19403](https://github.com/ant-design/ant-design/pull/19403)
+- 💄 Add `@modal-header-color` variable. [#19385](https://github.com/ant-design/ant-design/pull/19385)
+- 🛠 Multiple definitions of TypeScript. [#19377](https://github.com/ant-design/ant-design/pull/19377) [#19343](https://github.com/ant-design/ant-design/pull/19343) [#19333](https://github.com/ant-design/ant-design/pull/19333)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Improved some TypeScript definitions.

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
 
+## 3.24.3
+
+`2019-10-26`
+
+- 🐞 修复 Grid 响应式 `gutter` 失效的问题。 [#19308](https://github.com/ant-design/ant-design/pull/19308)
+- 🐞 修复小号 Table 边框样式问题。 [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 修复 Dropdown 父级菜单 SubMenu 的选中样式。 [#19313](https://github.com/ant-design/ant-design/pull/19313)
+- 🐞 修复 Typography & Drawer & Statistic 不能被 webpack 去除 moment.js 依赖的问题。[#19317](https://github.com/ant-design/ant-design/pull/19317)
+- 🐞 修复 Input.Search 出现 non-boolean attribute 和 unique key 警告信息。[#19357](https://github.com/ant-design/ant-design/pull/19357)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 🐞 修复 Input.Search 出现 `non-boolean attribute` 和 `unique key` 警告信息。[#19357](https://github.com/ant-design/ant-design/pull/19357)
```

File: CHANGELOG.en-US.md [hunk 8]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+
+`2019-10-26`
+
+- 🐞 Fix Grid responsive `gutter` not working. [#19308](https://github.com/ant-design/ant-design/pull/19308)
+- 🐞 Fix small Table border style. [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 Fix Menu in Dropdown SubMenu selected style. [#19313](https://github.com/ant-design/ant-design/pull/19313)
+- 🐞 Fix Typography & Drawer & Statistic not shaking moment.js by webpack. [#19317](https://github.com/ant-design/ant-design/pull/19317)
+- 🐞 Fix Input.Search react warnings of non-boolean attribute and unique key. [#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 Fix Modal icon position when not use title. [#19419](https://github.com/ant-design/ant-design/pull/19419)
+- 💄 Fix the error cursor of the checkbox. [#19403](https://github.com/ant-design/ant-design/pull/19403)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 不通，按中文翻译一下。

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (chenshuai2144) [reply CHANGELOG.zh-CN.md]: 这个要翻译成英语好难，超纲了

Comment (afc163) [reply CHANGELOG.zh-CN.md]: 原来的英文够用了。

File: CHANGELOG.en-US.md [hunk 9]
```diff
@@ -21,13 +21,13 @@ timeline: true
 
 - 🐞 Fix Grid responsive `gutter` not working. [#19308](https://github.com/ant-design/ant-design/pull/19308)
 - 🐞 Fix small Table border style. [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
-- 🐞 Fix Menu in Dropdown SubMenu selected style. [#19313](https://github.com/ant-design/ant-design/pull/19313)
-- 🐞 Fix Typography & Drawer & Statistic not shaking moment.js by webpack. [#19317](https://github.com/ant-design/ant-design/pull/19317)
-- 🐞 Fix Input.Search react warnings of non-boolean attribute and unique key. [#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 Fix the problem that the Menu in Dropdown is not highlighted when the SubMenu is selected. [#19313](https://github.com/ant-design/ant-design/pull/19313)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: the 不要滥用，后面两个 the 都可以去掉。

Comment (afc163) [reply CHANGELOG.en-US.md]: 第一个 the 改成 a

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/19404

File: CHANGELOG.zh-CN.md [hunk 10]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 3.24.3
+
+`2019-10-26`
+
+- 🐞 修复 Grid 响应式 `gutter` 失效的问题。 [#19308](https://github.com/ant-design/ant-design/pull/19308)
+- 🐞 修复小号 Table 边框样式问题。 [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 中文句号后面不需要额外空格。

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -15,6 +15,21 @@ timeline: true
...
+- 🐞 Fix small Table border style. [#19312](https://github.com/ant-design/ant-design/pull/19312) [#19342](https://github.com/ant-design/ant-design/pull/19342)
+- 🐞 Fix the problem that Menu in Dropdown is not highlighted when SubMenu is selected. [#19313](https://github.com/ant-design/ant-design/pull/19313)
+- 🐞 Fix the problem that moment.js in Typography & Drawer & Statistic can't be tree-shaking. [#19317](https://github.com/ant-design/ant-design/pull/19317)
+- 🐞 Fix Input.Search react warnings of `non-boolean attribute` and `unique key`. [#19357](https://github.com/ant-design/ant-design/pull/19357)
+- 🐞 Fix Modal icon position when not use title. [#19419](https://github.com/ant-design/ant-design/pull/19419)
+- 💄 Fix a problem with the mouse pointer error when Checkbox is disabled. [#19403](https://github.com/ant-design/ant-design/pull/19403)
+- 💄 Add `@modal-header-color` variable. [#19385](https://github.com/ant-design/ant-design/pull/19385)
+- 🛠 Improved some TypeScript definitions. [#19377](https://github.com/ant-design/ant-design/pull/19377) [#19343](https://github.com/ant-design/ant-design/pull/19343) [#19333](https://github.com/ant-design/ant-design/pull/19333)
```
Comment (shaodahong) [inline CHANGELOG.en-US.md]: @afc163 这样写感觉没什么意义, 最好表明更新了哪些定义, 至少我看库的 changelog 都是 **Cmd + F** 直接搜的

Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 大部分人不关注ts 的类型更新的。

**Pull request merged**